CHANGELOG.md

```diff
@@ -33,6 +33,12 @@
   ```
   
   In Dart SDK `>=1.25.0` this can be relaxed as `part of` can refer to a path.
+  To opt-in, `GeneratorBuilder` now has a new flag, `requireLibraryDirective`.
```

Comment:
Just an idea, but you can check the SDK version using `Platform.version` from `dart:io`, and we could make the default be based on that even? That would also likely require a dependency on `pub_semver`  but that should be ok.